### PR TITLE
Fix issues, refactor stuff

### PR DESCRIPTION
**Pull request by JustOptimize**

### CODE REVIEW

File: src/api/index.ts [hunk 1]
```diff
@@ -1,22 +1,22 @@
...
     const final: () => any = new Function("return () => " + path)();
     Object.defineProperty(final, "path", {
         value: path,
     });
     return final;
 }
 
-export function pathToFunction2(object: string, property: string) {
-    const final: () => any = new Function(`return () => ${object}.${property}`)();
+export function getFunctionByObjectAndProperty(objectName: string, property: string) {
```
Comment (Davilarek) [inline src/api/index.ts:13]: it doesn't get a function by object and property, it's more like a fake function resolver that resolves to object and property that *should* contain the real function after compilation

File: package.json [hunk 2]
```diff
@@ -7,9 +7,9 @@
     "discord-mod-compiler": "dist/cli.js"
   },
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1",
-    "cli": "node dist/cli.js",
-    "cli-debug": "node --inspect dist/cli.js",
+    "test": "npm run cli ./test/SampleBD.plugin.js RePlugged",
```
Comment (Davilarek) [inline package.json:10]: Don't use `SampleBD.plugin.js`; it's a placeholder
you probably meant to type `npm run cli ./test/sample/index.js RePlugged`

**Review by Davilarek — CHANGES_REQUESTED** [review]

File: src/api/index.ts (hunk 1 above)
Comment (JustOptimize) [reply src/api/index.ts:13]: yean, didn't know how to call it, was just sure that `pathToFunction2` wasn't the right name hahaha

**Review by Davilarek — APPROVED** [review]

**Comment by Davilarek** [comment]
lgtm

**Pull request merged**